### ISSUE TITLE
Can't connect Hue Bridge Emulator to Hue App on newest iOS update (3.2.0)

### ISSUE THREAD

**Issue by maxakuru** [mariusmotea/diyHue#306]
Similar to issue #284 

### Background
Didn't realize automatic updates were on for applications, and this morning my iPhone updated the Philips Hue app to 3.2.0. It seems this version breaks the connection of the Bridge to the App again. 

### Issue
The app identifies it as a Bridge, but when attempting to connect and GETing the `/hue/linkbutton` route, no ssdp requests are seen in the HueEmulator logging. 

### Things I've tried
- reinstalling Hue App
- restarting router/app/Hue Bridge/HueEmulator
- fresh install of HueEmulator

I've learned my lesson with automatic updates 😞

**Comment by dfloer** [comment]
Yeah, that sounds like the HTTPS issue. The version of the emulator here with nginx providing HTTP termination should work. Does it not? And did 3.1.0 work with nginx?

**Comment by mariusmotea** [comment]
Hi,

Is pretty important to know if the issue strike on nginx based version or on older release. This is because https issue was random discovered and maybe Philips enforce again the certificate validation in version 3.2.0. In this morning i made new commits witch will remove nginx as frontend because there is better ssl support in python native https server.
My recommendation is to perform these:
 - delete /opt/hue-emulator/cert.pem file
 - backup in any case the current configuration (/opt/hue-emulator/config.json
 - execute again the automatic installation script
 - if the application don't pair just delete the application data and try again

**Comment by fr8tra1n** [comment]
I can confirm 3.2.0 on iOS 11.4.1 does now work with the latest hue-emulator. If you encounter the linkbutton not working as expected, please try the last step in Mariusmotea's recommendation: delete the app, reinstall and try again.
@mariusmotea: 🥇 thank you for your hard work and excellent support!!
Additionally, timers and routines now work as well and do not crash the Hue app any longer. Nice.

**Comment by maxakuru** [comment]
Yep, working again. Thanks @mariusmotea !

Another question, which order should I set up diyHue Bridge and my Philips Hue Bridge? The first install I performed worked flawlessly, I had my Hue Bridge setup with all rooms and lights added, after linking the diyHue Bridge, it picked up all those settings. After factory reseting the Hue Bridge and readding my lights and rooms, I setup diyHue Bridge, but those settings didn't transfer over this time. Tried again starting with diyHue, again didn't pick them up.

Is order important here?
